src/components/main/PortalD/PortalD.js

```diff
@@ -33,6 +33,7 @@ import StationMap from '../../maps/StationMap';
 import AdjustableColumns from '../../util/AdjustableColumns';
 import capitalize from 'react-bootstrap/lib/utils/capitalize';
 import baseMaps from '../../maps/baseMaps';
+import NetworksMetadata from '../../info/NetworksMetadata';
 
```

Comment:
Maybe add one line comments to PortalD.js and PortalE.js explaining whichone does what? Perhaps it would be obvious if I was actually working on the code, but as-is, I find the names a little confusing.

Reply:
I renamed the components.